core/services/offchainreporting/run_saver.go

```diff
@@ -39,7 +39,7 @@ func (r *RunResultSaver) Start() error {
 			select {
 			case rr := <-r.runResults:
 				logger.Debugw("RunSaver: saving job run", "run", rr.Run, "task results", rr.TaskRunResults)
-				if _, err := r.pipelineRunner.InsertFinishedRunWithResults(context.Background(), rr.Run, rr.TaskRunResults); err != nil {
+				if _, err := r.pipelineRunner.InsertFinishedRunWithResults(context.Background(), rr.Run, rr.TaskRunResults, false); err != nil {
```

Comment:
Note - explicitly NOT saving the successful TaskRuns for OCR jobs here

Reply:
Do we want to leave this as a code comment?

Perhaps it could be clearer if instead of a boolean arg we used an enum so it can be named?